readme.md

```diff
@@ -74,6 +74,13 @@ Default: Auto-detected
 
 The registry URL is by default inferred from the npm defaults and `.npmrc`. This is beneficial as `package-json` and any project using it will work just like npm. This option is **only** intended for internal tools. You should **not** use this option in reusable packages. Prefer just using `.npmrc` whenever possible.
 
+##### agent
+
+Type: `object`
+
+Overwrite the `agent` that is passed down to [`got`](https://github.com/sindresorhus/got#agent).
+
```

Comment:
You should mention the use-case.

And don't mention Got. Just link to https://nodejs.org/api/http.html#http_http_request_url_options_callback

Reply:
Added the use-case.

`got` supports the additional behavior of "you can pass a map of agents" for the `agent` option, so I thought linking to that would be the right thing to do?

Reply:
Makes sense 👍